lib/Zonemaster/Backend/Validator.pm

```diff
@@ -211,6 +210,102 @@ sub jsonrpc_method {
     return joi->string->regex( $JSONRPC_METHOD_RE );
 }
 
+sub formats {
+    my ( $rpcapi ) = @_;
```

Comment:
I noticed that the only thing $rpcapi is ever used for is to read its config property. Have you considered making formats() take a Config object instead of an RPCAPI object?

Also, could you add some documentation for this procedure?

Reply:
Have you considered making formats() take a Config object instead of an RPCAPI object?
> Done

> Also, could you add some documentation for this procedure?
Also done.